applications/RANSApplication/CMakeLists.txt

```diff
@@ -10,11 +10,15 @@ include(pybind11Tools)
 include_directories( ${CMAKE_SOURCE_DIR}/kratos )
 include_directories( ${CMAKE_SOURCE_DIR}/applications/FluidDynamicsApplication )
 
-## RANSApplication Core sources
-file(GLOB_RECURSE KRATOS_RANS_APPLICATION_CORE
+## RANSApplication Core utilities sources
+file(GLOB_RECURSE KRATOS_RANS_APPLICATION_UTILITIES
     ${CMAKE_CURRENT_SOURCE_DIR}/rans_application.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/rans_application_variables.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/custom_utilities/*.cpp
+)
+
+## RANSApplication Core sources
+file(GLOB_RECURSE KRATOS_RANS_APPLICATION_CORE
     ${CMAKE_CURRENT_SOURCE_DIR}/custom_conditions/*.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/custom_elements/*.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/custom_processes/*.cpp
```

Comment:
```suggestion
## RANSApplication Core utilities sources
set(KRATOS_RANS_APPLICATION_CORE_INSTANCES
    ${CMAKE_CURRENT_SOURCE_DIR}/rans_application.cpp
    ${CMAKE_CURRENT_SOURCE_DIR}/rans_application_variables.cpp
)

## RANSApplication Core sources
file(GLOB_RECURSE KRATOS_RANS_APPLICATION_CORE
    ${CMAKE_CURRENT_SOURCE_DIR}/custom_utilities/*.cpp
    ${CMAKE_CURRENT_SOURCE_DIR}/custom_conditions/*.cpp
    ${CMAKE_CURRENT_SOURCE_DIR}/custom_elements/*.cpp
    ${CMAKE_CURRENT_SOURCE_DIR}/custom_processes/*.cpp
```

utilities folder should still remain in the former, otherwise the recurse can still cause something inside to be compiled before the instantiation (don't know if its the case right now but may prevent future problems)

Reply:
If I move it like you suggesting, it again fails with cotire :/

Reply:
I can put utiltiies in another LIST, so I know for sure those units are compiled first. Utilities should not depend on processes or elements or conditions or strategies as I think.